coverage/src/test/kotlin/org/rust/coverage/RsCoverageTest.kt

```diff
@@ -38,98 +37,119 @@ class RsCoverageTest : RunConfigurationTestBase() {
     override fun createRustupFixture(): RustupTestFixture = CoverageRustupTestFixture(project)
 
     fun `test main`() = doTest {
-        toml("Cargo.toml", """
+        toml(
```

Comment:
I don't think we want to change formatting here
Could you check similar changes in all changed files?